samples/boards/espressif/ulp/lp_core/debug_ulp/README.rst

```diff
@@ -0,0 +1,124 @@
+.. zephyr:code-sample:: debug-ulp
+   :name:
```

Comment:
Forgot the name :) prob a bug for me to fix that this didn't error out

Need to fix since this makes it quite hard to open the sample! https://builds.zephyrproject.io/zephyr/pr/84092/docs/samples/boards/espressif/ulp/lp_core/README.html